source/config/profileUpgradeSteps.py

```diff
@@ -84,3 +84,16 @@ def upgradeConfigFrom_4_to_5(profile):
 	except KeyError:
 		# Setting does not exist, no need for upgrade of this setting
 		log.debug("reportDetails not present, no action taken.")
+
+
+def upgradeConfigFrom_5_to_6(profile):

```

Comment:
Could we add type information?

```suggestion
def upgradeConfigFrom_5_to_6(profile: Dict):

```